### ISSUE TITLE
make essenceTrackIdentifier unbounded

### ISSUE THREAD

**Issue by MarcosSueiro** [WGBH/PBCore2.0#43]
Could this be done, along with the proposed essenceIdentifierType? Sometimes parts of an instantiation may come from different places, and it may help to provide a link to the original source.

**Comment by dmaccarn** [comment]
Would you not use the instantiationPart for this type of material?

**Comment by MarcosSueiro** [comment]
Ah yes, probably. Will this substitute essenceTrack?

**Comment by dmaccarn** [comment]
Not sure I understand.

**Comment by MarcosSueiro** [comment]
The suggestion is to allow for a repeatable essenceTrackIdentifier (along with essenceTrackIDType) to allow for more than one ID at the essenceTrack level. 

In our case, we may have an instantiation with, say, three essence tracks, named (by us) 67945.1A, 67945.1B, and 67945.1C. However, each of those files may also have a more descriptive file name used in our file management system: say, WNYC-LLSH-TrevorPinnock-1986-05-1-Cat#67945.1A. Furthermore, the file mangament system may assign a random UID. It would be nice to have the option of entering all the IDs that may be associated with other systems.

Another example would be the "bonus" section of a DVD including pictures which come from a different institution; it may be helpful to reference that institution's catalog number in an additional essenceTrackIdentifier field.

In short, this would serve the same purpose as the (repeatable) instantiation ID, but at the essenceTrack level.
